decidim-core/app/commands/decidim/create_omniauth_registration.rb

```diff
@@ -16,11 +16,19 @@ def initialize(form)
     #
     # Returns nothing.
     def call
-      return broadcast(:invalid) if form.invalid?
+      verify_oauth_signature!
 
       begin
-        create_user
-        create_identity
+        find_identity
```

Comment:
If you rename `find_identity` to `identity` (and use `||=`) you can just do:

`if identity.present?`